Providers/nxOMSAutomationWorker/automationworker/worker/jrdsclient.py

```diff
@@ -78,10 +88,85 @@ def get_sandbox_actions(self):
                 return None
 
             # success path
+            try:
+                #response.raw_data = response.raw_data.decode() if isinstance(response.raw_data, bytes) else response.raw_data
+                #sandbox_response = json.loads(response.raw_data)
+                #account_id = registration_response["PollingFrequency"]
+                #before return extract the header from response, and set polling freq
+
+                #if(response.deserialized_data.headers["cert rotation"]): #parse it to bool
+                #self.worker_certificate_rotation()
+                #print(response.headers)
+                import tracer
+                #tracer.log_worker_debug("debug logs are working! L96")
+                #tracer.log_worker_debug("response")
+                #tracer.log_worker_debug(dir(response))
+                #tracer.log_worker_debug("response.deserialized_data")
+                #tracer.log_worker_debug(response.deserialized_data)
+                #tracer.log_worker_debug("sandbox_response")
+                #tracer.log_worker_debug(sandbox_response)
+                #print response.headers
+                tracer.log_worker_debug("jrds107")
+
+                #Initiate certificate rotation if Headers is set True
+                tracer.log_worker_debug(workercertificaterotation.get_certificate_rotation_header_value())
+            
+                if(eval(workercertificaterotation.get_certificate_rotation_header_value())):
+                    tracer.log_worker_debug("Initiating certificate Rotation of Hybrid Worker")
+                    workercertificaterotation.set_certificate_rotation_header_value(DISABLE_CERT_ROTATION)  
+                    tracer.log_worker_debug("jrds114")
+                    tracer.log_worker_debug(workercertificaterotation.get_certificate_rotation_header_value())
+                    self.worker_certificate_rotation()
+            except Exception:
+                pass
```

Comment:
What will cause the exception. Do you want to log it ?

Reply:
exception should not be caused here, already given default values for headers